minimal/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "build": "cross-env NODE_ENV=production react-router build",
+    "dev": "react-router dev",
+    "start": "cross-env NODE_ENV=production react-router-serve ./build/server/index.js",
```

Comment:
If this is not marketed as "production ready" I might vote to remove the `NODE_ENV` stuff and the `cross-env` dependency

Reply:
Yeah, I agree. I actually don't know if we need any of this now with [this fix](https://github.com/remix-run/react-router/pull/12578)